engine/lib/phx/src/engine/event_bus.rs

```diff
@@ -0,0 +1,318 @@
+use mlua::{Function, Lua, Table};
+use std::{
```

Comment:
I prefer to keep `std` at the top, separated from other usages with empty line to make it more clear.